prestashoperpconnect/views/product_view.xml

```diff
@@ -91,7 +79,7 @@
         <record id="product_connector_presta_tree_view" model="ir.ui.view">
             <field name="model">prestashop.product.template</field>
             <field name="arch" type="xml">
-                <tree string="prestashop product" create="0">
+                <tree string="prestashop product">
```

Comment:
Don't remove the create="0". The goal is to not let create from there any record.